repose-aggregator/core/core-lib/src/main/java/org/openrepose/powerfilter/filtercontext/FilterContextFactory.java

```diff
@@ -104,7 +103,9 @@ private FilterContext loadFilterContext(Filter filter, Collection<EarClassLoader
             //Get the specific class to load from the application context
             Class c = filterClassLoader.loadClass(filterType.getFilterClass().getValue());
 
-            final javax.servlet.Filter newFilterInstance = (javax.servlet.Filter) filterContext.getBean(c);
+            //If Spring didn't load the filter as a bean, then try manually creating a new instance of the class
+            final javax.servlet.Filter newFilterInstance = (javax.servlet.Filter)
+                    (filterContext.getBeanNamesForType(c).length > 0 ? filterContext.getBean(c) : c.newInstance());
```

Comment:
Go ahead and stick with handling it as an exception instead of by bean name. We don't have full control of the bean name so i wouldn't count on it.